backend/tracim_backend/tests/__init__.py

```diff
@@ -97,7 +103,66 @@ def get_mailhog_mails(self) -> typing.List[typing.Any]:
         ).json()
 
 
-class FunctionalTest(unittest.TestCase):
+class TestWithHelpers(object):
+
+    session = None
```

Comment:
Where is filled this attribute ?

Reply:
This attribute is field by other object. In fact session is by default None (to avoid error somewhere in code) but this class TestWithHelpers is used by other class which set session.

Reply:
:ok_hand: